.travis.yml

```
@@ -86,14 +86,11 @@ matrix:
           env: PYTHON_VERSION=3.7 SETUP_CMD='bdist_conda'
                PACKAGING_TEST=true
 
-        # Older Python versions
-        - os: linux
-          env: PYTHON_VERSION=3.5 SETUP_CMD='test -V'
-
         # Test with Astropy dev and LTS versions
         - os: linux
-          env: PYTHON_VERSION=3.5 ASTROPY_VERSION=lts SETUP_CMD='test -V'
+          env: PYTHON_VERSION=3.6 ASTROPY_VERSION=lts SETUP_CMD='test -V'
                CONDA_DEPENDENCIES=$CONDA_DEPENDENCIES
+               PYTEST_VERSION<3.7
```

```suggestion
               PYTEST_VERSION='<3.7'
```

my bad for the syntax, sorry about that. Alternatively you can remove `pytest-cov` from this file altogether, `ci-helpers` should handle installing it for when needed.

---

I followed your suggestion in e146f22 .
Looks like the pytest-cov issue is resolved.